release/benchmarks/README.md

```diff
@@ -2,14 +2,17 @@
 
 ## Distributed Benchmarks
 
-All distributed tests are run on 64 nodes with 64 cores/node. Maximum number of nodes is achieved by adding 4 core nodes.
-
-| Dimension                                       | Quantity |
-| ---------                                       | -------- |
-| # nodes in cluster (with trivial task workload) | 2k+      |
-| # actors in cluster (with trivial workload)     | 40k+     |
```

Comment:
is this a regression?

Reply:
I think there is a regression from 20k -> 15k for actors. I feel 40k seems a broken number since each worker will have 3 sockets to gcs and we can at most have 65k sockets.

Reply:
From when this regression happens? Is it from 2.6? If it was already regressed, maybe we need some process improvements here to track the number better